Traceback (most recent call last):
  File "c:\Users\user\Desktop\hackathon_appss\hackathon_app\main.py", line 28, in <module>
    Demo_App().run()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\app.py", line 950, in run
    runTouchApp()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\base.py", line 582, in runTouchApp
    EventLoop.mainloop()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\base.py", line 347, in mainloop
    self.idle()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\base.py", line 399, in idle
    Clock.tick_draw()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\clock.py", line 783, in tick_draw
    self._process_events_before_frame()
  File "kivy\_clock.pyx", line 658, in kivy._clock.CyClockBase._process_events_before_frame
  File "kivy\_clock.pyx", line 701, in kivy._clock.CyClockBase._process_events_before_frame
  File "kivy\_clock.pyx", line 697, in kivy._clock.CyClockBase._process_events_before_frame
  File "kivy\_clock.pyx", line 218, in kivy._clock.ClockEvent.tick
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\uix\video.py", line 191, in _do_video_load
    self._video.play()
  File "C:\Users\user\anaconda3\lib\site-packages\kivy\core\video\video_gstplayer.py", line 94, in play
    self.player.set_volume(self.volume)
AttributeError: 'NoneType' object has no attribute 'set_volume'
